packages/api/tests/endpoints/test-rules.js

```diff
@@ -414,6 +626,11 @@ test('PUT replaces a rule', async (t) => {
     .set('Authorization', `Bearer ${jwtAuthToken}`)
     .expect(200);
 
+  const dbRecords = await dbClient.first()
+    .from(tableNames.rules)
+    .where({ name: expectedRule.name });
+
+  t.not(dbRecords.queue_url, undefined);
```

Comment:
I admit that I have a really hard time following this test's logic, but I **think** we should be testing the opposite and verifying that `queue_url` **is undefined**. 

The test creates the `expectedRule` using the PUT endpoint, and that rule has the `queueUrl` property removed from it. Thus, the `deepEqual` below proves that the `actualRule` returned after the `PUT` no longer has a `queueUrl` property in Dynamo, and nor should it have a `queue_url` property in Postgres. Unless I'm misunderstanding, which is quite possible

Reply:
Yeah, actually your interpretation is one hundred percent correct. It _should_ be checking that `queue_url` is not defined on the postgres record. I think what I meant to check was that it initially had a value for `queue_url` prior to the update call, but that doesn't make sense because I don't insert `testRule` into the pg table. Let me fix this.

Reply:
[`717d0e5` (#2012)](https://github.com/nasa/cumulus/pull/2012/commits/717d0e50dc816dd503b437d92a583a563a6c0766)